Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_aitdm.nut

```diff
@@ -460,7 +460,18 @@ void function SquadHandler( array<entity> guys )
 		// Get point and send our whole squad to it
 		points = GetNPCArrayOfEnemies( team )
 		if ( points.len() == 0 ) // can't find any points here
+		{
+			// Have to wait some amount of time before continuing
+			// because if we don't the server will continue checking this
+			// forever, aren't loops fun?
+			// This definitely didn't waste ~8 hours of my time reverting various
```

Comment:
+ 2 weeks of project time
+ at least 50 hours of time between gecko, blue, and I...